tests/unit/hive_metastore/test_tables.py

```diff
@@ -137,37 +137,60 @@ def test_tables_returning_error_when_describing():
 
 
 def test_is_dbfs_root():
```

Comment:
can you do `@pytest.mark.parametrise` here?

Reply:
To reduce redundancy?

Reply:
Sure will do.